README.md

```diff
@@ -164,7 +145,7 @@ Supported Platforms
 
 ### sshcommand_version
 
-- default: `0.9.0`
+- default: `0.8.0`
```

Comment:
I think this got updated in a separate PR and was never updated in the `defaults/main.yml.base` file, mind verifying/fixing if so?

Reply:
Of course I noticed it only after I pushed.
Those last-minute pushes before leaving the office ... ;-)